### ISSUE TITLE
not able to disable shuffling using : make_torch_dataloader

### ISSUE THREAD

**Issue by Warra07** [uber/petastorm#721]
I'm not sure i understood the way the shuffling work using make_torch_dataloader, i would like to disable shuffling on prediction but it doesn't seem to work as expected, here is how i create the data loader: 
```python
      converter = make_spark_converter(df)
      return converter.make_torch_dataloader(transform_spec= TransformSpec(...),  batch_size=self.batch_size,num_epochs=1,
                                            shuffle_row_groups=False, shuffling_queue_capacity= 0, workers_count= 1) ```

but when i iterate over my dataset, the first batch of data doesn't correspond to the head of my dataframe (sometime it does ? ?)
Am i doing something wrong ?

**Comment by selitvin** [comment]
`make_spark_converter` would persist the dataframe into a parquet store. This is done by multiple spark executors in parallel and the data is written into multiple files. Due to the parallel nature of the writes, I am not even sure if a mechanism to get predictable order of rows exists. Would need to read more about it.

**Comment by Warra07** [comment]
Well my current (dumb) solution is simply adding an incrementing id to my df and use it to reorder.
I guess a solution would be to use only one spark executor, but i think it would be a bad idea performance wise. i'd prefer using an id and maybe merge with initial dataframe if needed.
